cmake/RunTests.cmake

```diff
@@ -34,6 +37,9 @@ execute_process(
   RESULT_VARIABLE res
   ${EXTRA_ARGS})
 
+file(REMOVE ${WORKING_DIR}/Xtest_rplugin_manifest)
+file(REMOVE_RECURSE ${WORKING_DIR}/Xtest_xdg)
```

Comment:
All I know is it does what I expected it to do.  I got it from here: https://cmake.org/cmake/help/v3.0/command/file.html